example/linked_list/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+#include_directories(${CMAKE_CURRENT_LIST_DIR})
+
+macro(ADD_EXAMPLE_INCLUDE name)
+    add_executable(example_${name} example_${name}.f90)
+    target_include_drectories(example_${name} ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
@arjenmarkus 
By removing this line (with `target_include_directories`) and by added `stdlib_child_list.f90` and `stdlib_linked_list.f90` in the file `src/CMakeLists.txt`, I could compile succesfully stdlib (including the examples)
```suggestion
```